src/components/Profile/components/Identification/index.tsx

```diff
@@ -692,13 +692,36 @@ const Identification = ({ profile, myProf, isOnline, createSnackbar, fetchProfil
                   xs={12}
                   className={styles.identification_card_content_card_container_info_name}
                 >
-                  <Typography variant="h6" paragraph>
-                    {`${
-                      profile.Title && profile.PersonType === 'fac' ? `${profile.Title} ` : ''
-                    }${profile.FirstName}${hasNickname ? ` (${profile.NickName})` : ''} ${
-                      profile.LastName
-                    }${hasMaidenName ? ` (${profile.MaidenName})` : ''}`}
-                  </Typography>
+
+                  <div style={{ display: 'flex', alignItems: 'center', gap: '8px' }}>
+                    <Typography variant="h6" paragraph style={{ marginBottom: 0 }}>
+                      {`${
+                        userProfile.Title && userProfile.PersonType === 'fac'
```

Comment:
Is this right?  I think the original change was replacing useprofile with profile, and this looks like it's undoing that.  Github is generating some warnings about this.